apps/docs/src/pages/login.tsx

```
@@ -117,9 +117,8 @@ export default function LoginPage(): React.ReactElement {
                     )}
                     <p className={styles.configHelp}>
                       <strong>For admins:</strong> Set these in your hosting
-                      platform's environment variables (Netlify: Site
-                      Configuration → Environment Variables). A new deploy is
-                      required after adding them.
+                      platform's environment variables (Azure Static Web Apps,
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)